quiche/src/recovery/mod.rs

```diff
@@ -251,6 +253,97 @@ impl Recovery {
     }
 }
 
+/// BBR settings used to customize the algorithm's behavior.
+///
+/// This is an experimental API.
+///
+/// A congestion control algorithm has dual-responsibility of effective network
+/// utilization and avoiding congestion. Custom values should be choosen
+/// carefully since incorrect values can lead to network degradation for all
+/// connections on the shared network.
+#[derive(Debug, Default, Copy, Clone, PartialEq)]
+#[repr(C)]
```

Comment:
I had mentioned initial RTT to you as something we may want to override at this level.  But maybe we can avoid paying attention to the RTT value we use in initialization.

Reply:
I was focusing on the gcongestion BBR params in this PR. I definitely think its worth experimenting with rtt, but I assume that will also affect other CC algorithms.

Reply:
This is the [initial_rtt](https://github.com/cloudflare/quiche/pull/2019/commits/56dfecf10c4c6e7099c0fba56609f9fda770d335) override, only scoped to  gcongestion.